minigalaxy/ui/library.py

```diff
@@ -95,7 +95,8 @@ def __create_gametiles(self) -> None:
 
         for game in self.games:
             if game not in games_with_tiles:
-                self.__add_gametile(game)
+                if Config.get("show_hidden_games") or not game.get_info("hide_game"):
+                        self.__add_gametile(game)
```

Comment:
Please put proper implementation in __filter_library_func() instead of __create_gametiles()